src/core/evaluator.js

```diff
@@ -1606,9 +1618,10 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
               var name = args[0].name;
               if (xobjsCache.key === name) {
-                if (xobjsCache.texts) {
-                  Util.appendToArray(textContent.items, xobjsCache.texts.items);
-                  Util.extendObj(textContent.styles, xobjsCache.texts.styles);
+                if (xobjsCache.cachedItems) {
+                  xobjsCache.cachedItems.forEach(function(chunk) {
```

Comment:
Since it's just items use `sink.enqueue({ items: xobjsCache.cachedItems, styles: {}, }, xobjsCache.cachedItems.length);` (no iterating)